.gitignore

```diff
@@ -195,6 +195,8 @@ uptime-*.json
 /src/lib/libtor-meminfo-testing.a
 /src/lib/libtor-net.a
 /src/lib/libtor-net-testing.a
+/src/lib/libtor-osinfo.a
+/src/lib/libtor-osinfo-testing.a
```

Comment:
So here is an idea. How about we have an "os" library for which we could put all the things we use to interact with the operating system.

I'm thinking about this `get_uname()` but also, set file desc, get available memory, etc... ?

Reply:
Hm. I'm not so sure there.  Isn't *everything* that ends in a syscall "something that interacts with the operating system?"